tfoptests/ops.py

```diff
@@ -1227,8 +1230,70 @@ def execute_fake_quant_with_min_max_vars_per_channel(self):
     def execute_check_numerics(self):
         return [tf.debugging.check_numerics(tensor=self.vars[0], message=self.op["message"])]
 
+    def execute_adjust_contrast(self):
+        return [tf.image.adjust_contrast(self.vars[0],self.vars[1])]
+
+    def execute_adjust_contrast_v2(self):
+        return [tf.compat.v2.image.adjust_contrast(self.vars[0],self.vars[1])]
+
     def execute_strings_split(self):
         print("strings.split input: ", self.vars[0])
         out = tf.strings.split(self.vars[0], sep=self.op["split"])
         print("strings.split output: ", out)
         return [out]
+
+    def execute_bitcast(self):
+        return [tf.bitcast(self.vars[0], self.op["output"])]
+
+    def execute_bitwise_or(self):
+        return [tf.bitwise.bitwise_or(self.vars[0], self.vars[1])]
+
+    def execute_bitwise_xor(self):
+        return [tf.bitwise.bitwise_xor(self.vars[0], self.vars[1])]
+
+    def execute_crop_and_resize(self):
+        return [tf.image.crop_and_resize(image = self.vars[0], boxes = self.vars[1], crop_size = self.vars[2], box_ind = self.vars[3])]
+
+    def execute_draw_bounding_boxes(self):
+        return [tf.image.draw_bounding_boxes(images = self.vars[0], boxes = self.vars[1])]
+
+    def execute_resize_bilinear(self):
+        return [tf.image.resize_bilinear(images = self.vars[0], size = self.vars[1])]
+
+    def execute_resize_nearest_neighbor(self):
+        return [tf.image.resize_nearest_neighbor(images = self.vars[0], size = self.vars[1])]
+
+    def execute_non_max_suppression(self):
+        return [tf.image.non_max_suppression(boxes = self.vars[0], scores = self.vars[1], max_output_size = self.vars[2],
+                                             iou_threshold=0.5, score_threshold=float('-inf'))]
```

Comment:
You have test cases defined with 5 inputs, but only use 3 of them here - i.e., threshold args are ignored.
Might be easier to do it for attributes with "dict" style (like your log softmax axis) for float attributes.

If we want these extra args to be optional (i.e., don't have to define them in every case) we can do something like this:
https://github.com/KonduitAI/ImportTests/blob/6c032dacc431b3ff86d41323161699d0f16497f7/tfoptests/ops.py#L749-L751